backend/users/migrations/0001_initial.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.1.7 on 2019-02-19 23:51
```

Comment:
I think this migration can be deleted.

Reply:
We will push it now with the edits that we have made. If you want us to remove this, we can push it again when you're here. (We don't want to delete it and then end up needing it!)